internal/providers/mailgun.provider.go

```
@@ -28,32 +30,40 @@ func NewMailgun() *Mailgun {
+		HTML:    content,
 	}
 
-	json, errJSON := json.Marshal(emailPayload)
-	if errJSON != nil {
-		return errJSON
+	json, errJ := json.Marshal(body)
+	if errJ != nil {
+		return errJ
 	}
 
 	req, errReq := http.NewRequest("POST", e.url, bytes.NewBuffer(json))
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)